openapi/routerinfo_middleware.go

```diff
@@ -0,0 +1,64 @@
+// Copyright 2021 Mia srl
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package openapi
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+
+	"github.com/rond-authz/rond/internal/utils"
+
+	"github.com/gorilla/mux"
+	"github.com/sirupsen/logrus"
+)
+
+type RouterInfoKey struct{}
```

Comment:
Why is it public? This could be private of the package

Reply:
It was already exported, however this is the main package therefore the exported/private makes no difference.

Reply:
It is the openapi package, why the main package?

Reply:
ops, didn't notice lol

Reply:
I checked and, for now this key cannot be made private because it is used in tests in the core package; i'm adding a TODO comment to make this private in the future